qa-tests-backend/src/test/groovy/ClientCertAuthTest.groovy

```diff
@@ -41,7 +41,7 @@ class ClientCertAuthTest extends BaseSpecification {
             log.info "Client cert auth provider ID is ${providerIDs[i]}"
             GroupService.addDefaultMapping(providerIDs[i], "Continuous Integration")
             certTokens[i] = AuthProviderService.getAuthProviderLoginToken(providerIDs[i])
-            log.info "Certificate token is ${certTokens[i]}"
+            log.info "Certificate token is ${certTokens[i]}  # nosecret"
```

Comment:
I guess we have a small typo here. Based on defined ignore patterns, it should be `notsecret`.
```suggestion
            log.info "Certificate token is ${certTokens[i]} #notsecret"
```